### DISCUSSION TITLE
Text is disappearing after selection

### DISCUSSION THREAD

**Discussion by Dolorm** [slint-ui/slint#4453] [Q&A]
Hi!
I'm working on a project where I want user to copy result text from TextEdit. But it disappears when I try to select it. I tried it on slint examples where there's TextEdit or LineEdit and I had the same issue. I only started using slint, so is there a way to handle this?
OS: Linux/Debian
[2024-01-25-21-43-57.webm](https://github.com/slint-ui/slint/assets/108493813/8b4e8d4d-9093-402c-837d-b96e57286cb9)

**Comment by ogoffart** [comment]
Is that Plasma with Wayland.
Then it should be fixed with https://github.com/slint-ui/slint/pull/4188
